__tests__/helpers/BasicPointsAndBadges/ConnectExternalAccountActivity.test.ts

```diff
@@ -0,0 +1,98 @@
+import { awardConnectExternalAccountActivityPoints, ConnectExternalAccountActivity } from "../../../src/helpers/BasicPointsAndBadges/ConnectExternalAccountActivity";
+
+jest.mock('@careevolution/mydatahelps-js', () => {
+
+    const externalAccounts = [{
+        "id" : 1,
+        "provider": { "id": 1, "name": "Garmin", "category": "Device Manufacturer"} 
+    },
+    {
+        "id" : 2,
+        "provider": { "id": 2, "name": "FitBit", "category": "Device Manufacturer"} 
+    },
+    {
+        "id" : 3,
+        "provider": { "id": 3, "name": "Health Plan 1", "category": "Health Plan"} 
+    }];
+
+    return {
+        __esModule: true,
+        default: {
+            token: { access_token: '1' },
+            getCurrentLanguage: () => jest.fn(),
+            getExternalAccounts: jest.fn(() => {
+                return Promise.resolve(externalAccounts);
+            }),
+            on: jest.fn(),
+            off: jest.fn(),
+        },
+        MyDataHelps: {
```

Comment:
I believe you can remove this `MyDataHelps` entry.  I'm pretty sure the MDH-JS module doesn't export anything named `MyDataHelps`.  It just exports it as the `default`.